src/main/java/frc/robot/commands/FlyWheelCommands/ShootNoteCommand.java

```diff
@@ -0,0 +1,24 @@
+package frc.robot.commands.FlyWheelCommands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.FlyWheelSubsystem;
+
+public class ShootNoteCommand extends Command {
+    FlyWheelSubsystem flyWheel;
+
+    public ShootNoteCommand(FlyWheelSubsystem flyWheel){
+        this.flyWheel = flyWheel;
+    }
+
+    @Override
+    public void execute(){
+        flyWheel.rampUp();
+    }
+
+    @Override
+    public void end(boolean i){
+        flyWheel.rampDown();
+    }
```

Comment:
Is the plan here to execute this command with a `whileTrue`?

Reply:
yes, that is what I was thinking, do you think it should be a on true statement?

Reply:
I think `whileTrue` makes sense to use here!